tests/minimal.wcon

```diff
@@ -0,0 +1 @@
+{"tracker-commons":true, "units":{}, "data":[]}
```

Comment:
I think we require units on "x", "y",  and "t".  That we don't use them in the data doesn't enter the specification.